### PR TITLE
Use angular

### PR DESCRIPTION
**Pull request by Harman-B**
Migrating the code to angular

### CODE REVIEW

**Comment by Harman-B** [comment]
we can keep/ delete the original javascript code in files gitplay.js, gitplay.html and gitplay.css

**Comment by devilpreet** [comment]
Wo hoo!! Version 0.2 in angular is out !!
No need to keep original javascript code files. As they are safe in the history of the project. Viewing an older commit/file is easy task. Keeping unused code is generally bad practice.
Also, usually when the branch is merged back, it is deleted as now it is contained in master. Keeping number of branches low ( or equal to parallel developments) is good practise
I would add comments in this pull request. In case you want them going forward > Open merge request, mark reviewer and comment "ready to merge".

File: gitplay/.gitignore [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```
Comment (devilpreet) [inline gitplay/.gitignore:1]: Seems like you picked up a template. Nice!!

Comment (Harman-B) [reply gitplay/.gitignore:1]: This was not me. This file was auto generated by Angular. But now that you have pointed it out. I will try to understand it.

**Comment by Harman-B** [comment]
Deleting merged branch.

**Pull request merged**